packages/api/tests/models/granules/test-granule-reporting.js

```diff
@@ -1,6 +1,7 @@
 const test = require('ava');
 const sinon = require('sinon');
 
+const S3 = require('@cumulus/aws-client/S3');
```

Comment:
This seems like more than bootstrap time improvement?

Reply:
This is unrelated to bootstrap time.

This test was failing intermittently for me because it was timing out. The test creates a couple granules with a dummy file in each of them. Those files were not created in S3 so that, when the code under test tried to get the size of those files from S3, it was retrying for 7 seconds before failing.

By staging the dummy files to S3, the "get size" operation returns immediately and took the test time from 10+ seconds and intermittent failures to working consistently in milliseconds.